corehq/apps/case_search/tests/test_ancestor_functions.py

```diff
@@ -162,3 +163,15 @@ def test_nested_ancestor_exists(self):
             {'_xpath_query': xpath},
         )
         self.assertItemsEqual(query1.get_ids(), ['a1'])
+
+    @generate_cases([
+        ("ancestor_exists(status='active' and subcase-exists('parent', city = 'LA'))",),
+        ("ancestor_exists(status='active' and subcase-count('parent', city = 'LA')) > 3",),
```

Comment:
I think these should be `ancestor-exists` (dash not underscore)

Reply:
thanks! [9eb4fc1](https://github.com/dimagi/commcare-hq/pull/33014/commits/9eb4fc14e79fbd181fcff0bcdd0d4abc4acef36e)